### PR TITLE
Upgrade to 1.6.1

### PR DESCRIPTION
**Pull request by LRFLEW**
I re-built the PKGBUILD file based on the community/benchmark package, so there are a number of non-functional changes to bring it closer to how that PKGBUILD file works.

### CODE REVIEW

File: PKGBUILD
Comment (janisozaur) [inline PKGBUILD:26]: Isn't it supposed to be release or a similar one?

**Review by janisozaur — COMMENTED** [review]
I'll have to figure out what do I need to do with this after merging

**Comment by janisozaur** [comment]
Also - surprised to see you here, @LRFLEW. Thanks

File: PKGBUILD
Comment (LRFLEW) [reply PKGBUILD:26]: I thought so too, but this is what the 64-bit Benchmark package is doing. I'm not sure why, though.

https://github.com/archlinux/svntogit-community/blob/packages/benchmark/trunk/PKGBUILD#L26

**Comment by LRFLEW** [comment]
@janisozaur Yeah, I just got a new laptop a few weeks ago, and I decided to install Arch on it (using KDE Plasma as my DE). I've also recently been experimenting with how different systems implement `rand()`, and after seeing FreeBSD's (old) implementation (which is also used by Apple), I wanted to test the performance of some functions on 32-bit vs 64-bit. I ran into some trouble with this AUR package, so I ended up making this version for myself, and creating the PR here since I already did the work. (Maybe I'll make a patch suggestion to FreeBSD based on my findings.)

File: PKGBUILD
Comment (janisozaur) [reply PKGBUILD:26]: I've collected some information and I believe this is a bug in packaging. I've filed a bug: https://bugs.archlinux.org/task/73185 and notified maintainers. Let's wait until they respond.

**Review by janisozaur — APPROVED** [review]

**Comment by janisozaur** [comment]
They rely on `makepkg.conf` to provide optimisations and so the code is correct. I've submitted the updated package to AUR

**Pull request merged**
